typify-impl/src/structs.rs

```diff
@@ -373,12 +373,12 @@ pub(crate) fn generate_serde_attr(
     let default_fn = match (state, &prop_type.details) {
         (StructPropertyState::Optional, TypeEntryDetails::Option(_)) => {
             serde_options.push(quote! { default });
-            serde_options.push(quote! { skip_serializing_if = "Option::is_none" });
+            serde_options.push(quote! { skip_serializing_if = "std::option::Option::is_none" });
```

Comment:
Should we prefix with :: ?

Reply:
Good catch, yes we should.  There's also a `std::vec::Vec` right below it which also needs to be fully qualified.